symfony/client/src/orangehrmAdminPlugin/pages/payGrade/SavePayCurrency.vue

```diff
@@ -0,0 +1,148 @@
+ * all the essential functionalities required for any enterprise.
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+ -->
+<template>
+  <div class="orangehrm-card-container">
+    <oxd-text tag="h6" class="orangehrm-main-title">Add Currency</oxd-text>
+    <oxd-divider />
+    <oxd-form :loading="isLoading" @submitValid="onSave">
+      <oxd-form-row>
+        <oxd-grid :cols="1" class="orangehrm-full-width-grid">
+          <oxd-grid-item>
+            <oxd-input-field
+              type="dropdown"
+              label="Currency"
+              v-model="payCurrency.currencyId"
+              :rules="rules.currencyId"
+              :create-options="loadOptions"
+              :lazyLoad="false"
+              :clear="false"
+              required
+            />
+          </oxd-grid-item>
+        </oxd-grid>
+      </oxd-form-row>
+      <oxd-form-row>
+        <oxd-grid :cols="2" class="orangehrm-full-width-grid">
+          <oxd-grid-item>
+            <oxd-input-field
+              label="Minimum Salary"
+              v-model="payCurrency.minSalary"
+              :rules="rules.minSalary"
+            />
+          </oxd-grid-item>
+          <oxd-grid-item>
+            <oxd-input-field
+              label="Maximum Salary"
+              v-model="payCurrency.maxSalary"
+              :rules="rules.maxSalary"
+            />
+          </oxd-grid-item>
+        </oxd-grid>
+      </oxd-form-row>
+      <oxd-divider />
+
+      <oxd-form-actions>
+        <required-text />
+        <oxd-button
+          type="button"
+          displayType="ghost"
+          label="Cancel"
+          @click="onCancel"
+        />
+        <submit-button />
+      </oxd-form-actions>
+    </oxd-form>
+  </div>
+</template>
+
+<script>
+import {APIService} from '@/core/util/services/api.service';
+import {required, max, digitsOnly} from '@orangehrm/core/util/validation/rules';
+const payCurrencyModel = {
+  currencyId: '',
+  minSalary: '',
+  maxSalary: '',
+};
+export default {
+  name: 'save-pay-currency',
+  props: {
+    payGradeId: {
+      type: String,
+      required: true,
+    },
+  },
+  setup(props) {
+    const httpAllowed = new APIService(
```

Comment:
No need to initiate a new api service. refer [this comment](https://github.com/orangehrm/orangehrm/pull/859/files#r680691777)

Reply:
Done